src/main/java/teammates/ui/webapi/FeedbackSessionRemindEmailWorkerAction.java

```diff
@@ -32,7 +32,7 @@ public JsonResult execute() {
 
             List<StudentAttributes> studentsToRemindList = new ArrayList<>();
             for (StudentAttributes student : studentList) {
-                if (!logic.isFeedbackSessionCompletedByStudent(session, student.getEmail())) {
+                if (!logic.isFeedbackSessionCompletedByStudent(session, student.getEmail(), student.getTeam())) {
                     studentsToRemindList.add(student);
                 }
             }
```

Comment:
Can try to use stream for this part of code:)

```
List<StudentAttributes> studentsToRemindList = studentList.stream().filter(student -> 
        !logic.isFeedbackSessionCompletedByStudent(session, student.getEmail())
);
```

Reply:
Use of stream is encouraged, as it presents a cleaner, possibly immutable semantics. The use of `filter` means that we are getting the same set of data, possibly less, instead of potentially doing something to the data.